infra/src/main/kotlin/fi/fta/geoviite/infra/geometry/GeometryService.kt

```diff
@@ -291,9 +295,11 @@ class GeometryService @Autowired constructor(
                 )
             }
         val csvFileContent = locationTrackElementListingToCsv(trackNumberService.list(OFFICIAL), elementListing)
+        val dateFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy").withZone(ZoneId.of("Europe/Helsinki"))
```

Comment:
Näissä paras olisi kun tuo Europe/Helsinki olisi vain frontin tietämä asia (voi lähettää sen queryparamina pyytäessä näitä). Toki meillä ei toistaiseksi ole toista kieltä ja tästä tuo on melko helppo vaihtaa -> vain kosmeettinen juttu toistaiseksi